doc/rbd/iscsi-initiator-linux.rst

```diff
@@ -90,3 +90,9 @@ group for each path.
 
 You should now be able to use the RBD image like you would a normal
 multipath’d iSCSI disk.
+
+4. Logout from target:
+
+   ::
+
+      # iscsiadm -m node -T iqn.2003-01.org.linux-iscsi.rheln1 -p 192.168.56.101:3260 -u
```

Comment:
The man page says that `-p` is "only valid for discovery, or for node operations with the new operator".  Is it actually needed here?
```suggestion
      # iscsiadm -m node -T iqn.2003-01.org.linux-iscsi.rheln1 -u
```